contrib/kind.sh

```diff
@@ -886,6 +886,9 @@ install_metallb() {
   fi
   git clone https://github.com/metallb/metallb.git
   pushd metallb
+  # temporary fix for metallb issue
+  # https://github.com/metallb/metallb/commit/fdf92741c7fac20eedf3caa0aa922f9ff0f0e7dd#r110009241
```

Comment:
ouch, wondering if we should be cloning in a specific release of metallb https://github.com/metallb/metallb/commit/fdf92741c7fac20eedf3caa0aa922f9ff0f0e7dd#r110009241 instead of pulling from master: https://github.com/metallb/metallb/tree/v0.13.9